### PR TITLE
Publish symbols for the native XamlHost dll

### PR DESCRIPTION
**Pull request by ocalvo**
Fixes #

## PR Type
What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../readme.md#supported)
- [ ] Pull Request has been submitted to the documentation repository [instructions](..\contributing.md#docs). Link: 
- [ ] Sample in sample app has been added / updated (for bug fixes / features)
    - [ ] Icon has been created (if new sample) following the [Thumbnail Style Guide and templates](https://github.com/windows-toolkit/WindowsCommunityToolkit-design-assets)
- [ ] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Header has been added to all new source files (run *build/UpdateHeaders.bat*)
- [ ] Contains **NO** breaking changes

### CODE REVIEW

File: Microsoft.Toolkit.Sample.UWP.App/Microsoft.Toolkit.Sample.UWP.App.csproj [hunk 1]
```diff
@@ -19,6 +19,8 @@
     <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <WindowsXamlEnableOverview>true</WindowsXamlEnableOverview>
     <AppxPackageSigningEnabled>false</AppxPackageSigningEnabled>
+    <!--AssetTargetFallback>$(AssetTargetFallback);native0.0</AssetTargetFallback-->
```
Comment (ocalvo) [inline Microsoft.Toolkit.Sample.UWP.App/Microsoft.Toolkit.Sample.UWP.App.csproj:22]: Remove

File: Microsoft.Toolkit.Sample.UWP.App/Microsoft.Toolkit.Sample.UWP.App.csproj [hunk 2]
```diff
@@ -169,14 +171,16 @@
     </PackageReference>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.Toolkit.Win32.UI.XamlApplication\Microsoft.Toolkit.Win32.UI.XamlApplication.vcxproj">
+    <Reference Include="..\$(XamlApp-Platform)\$(Configuration)\Microsoft.Toolkit.Win32.UI.XamlApplication\Microsoft.Toolkit.Win32.UI.XamlHost.winmd" />
+    <!--ProjectReference Include="..\Microsoft.Toolkit.Win32.UI.XamlApplication\Microsoft.Toolkit.Win32.UI.XamlApplication.vcxproj">
```
Comment (ocalvo) [inline Microsoft.Toolkit.Sample.UWP.App/Microsoft.Toolkit.Sample.UWP.App.csproj:175]: Remove

File: Microsoft.Toolkit.Win32.UI.XamlApplication/Microsoft.Toolkit.Win32.UI.XamlApplication.vcxproj [hunk 3]
```diff
@@ -153,11 +156,12 @@
     <Import Project="..\packages\Microsoft.Windows.CppWinRT.2.0.190506.1\build\native\Microsoft.Windows.CppWinRT.targets" Condition="Exists('..\packages\Microsoft.Windows.CppWinRT.2.0.190506.1\build\native\Microsoft.Windows.CppWinRT.targets')" />
   </ImportGroup>
   <Target Name="Pack" />
```
Comment (ocalvo) [inline Microsoft.Toolkit.Win32.UI.XamlApplication/Microsoft.Toolkit.Win32.UI.XamlApplication.vcxproj:158]: Do we still need these hack?

**Review by ocalvo — COMMENTED** [review]
I cannot approve my own PR, but I only had minor comments

File: Microsoft.Toolkit.Sample.UWP.App/Microsoft.Toolkit.Sample.UWP.App.csproj (hunk 2 above)
Comment (kmahone) [reply Microsoft.Toolkit.Sample.UWP.App/Microsoft.Toolkit.Sample.UWP.App.csproj]: Why did you switch from ProjectReference to a Reference? I've switched it back to a ProjectReference which seems better. Is there a reason not to do that?

File: Microsoft.Toolkit.Win32.UI.XamlApplication/Microsoft.Toolkit.Win32.UI.XamlApplication.vcxproj (hunk 3 above)
Comment (kmahone) [reply Microsoft.Toolkit.Win32.UI.XamlApplication/Microsoft.Toolkit.Win32.UI.XamlApplication.vcxproj]: Looks like we still need them. I see build errors, if I remove them and try to run msbuild /pack

**Review by kmahone — APPROVED** [review]

**Pull request merged**